__tests__/components/editor/actions/PreviewButton.test.js

```diff
@@ -17,6 +17,9 @@ describe('When the button is clicked', () => {
         modal: {
           name: undefined,
         },
+        modal: {
```

Comment:
Why is modal in twice?

Reply:
I think that was introduced in a merge conflict.  I'll remove it.